src/views/ops/Template/Playbook/PlaybookDetail/Workspace.vue

```diff
@@ -34,7 +34,7 @@
             </el-tab-pane>
           </el-tabs>
           <div style="display: flex;margin-top:10px;justify-content: space-between" />
-          <el-form ref="form" label-position="left" label-width="30px">
+          <el-form v-if="!disableEdit" ref="form" label-position="left" label-width="30px">
             <div class="form-content">
               <el-form-item label="" prop="variable">
                 <Variable :value.sync="variables" @input="setVariable" />
```

Comment:
I apologize for the confusion, but there seems to be an error in your text as it doesn't seem formatted correctly. Can you please provide the actual code so I can analyze its differences and potential improvements?